extensions/2.0/Vendor/EXT_texture_astc/README.md

```diff
@@ -0,0 +1,236 @@
+<!--
+Copyright 2024 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# EXT_texture_astc
+
+## Contributors
+
+- Wasim Abbas [@wasimabbas-arm](https://github.com/wasimabbas-arm)
+- Jan-Harald Fredriksen [@janharaldfredriksen-arm](https://github.com/janharaldfredriksen-arm)
+- Peter Harris [@solidpixel](https://github.com/solidpixel)
+- Mark Callow [@MarkCallow](https://github.com/MarkCallow)
+- Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)
+
+Copyright 2024 The Khronos Group Inc.
```

Comment:
2025

Reply:
Fixed in https://github.com/KhronosGroup/glTF/pull/2369/commits/d1145f8f08440ae80ac407156d328d987c746557